README.md

```diff
@@ -69,84 +103,38 @@ Travis CI always has a build with a pinned version of Rustc matching the oldest
-    -   a `Standard` distribution for integers, floats,and derived types
-        including tuples, arrays and `Option`
-    -   unbiased sampling from specified `Range`s
-    -   sampling from exponential/normal/gamma distributions
-    -   sampling from binomial/poisson distributions
-    -   `gen_bool` aka Bernoulli distribution
--   `seq`-uence related functionality:
-    -   sampling a subset of elements
-    -   randomly shuffling a list
 
 ## Crate Features
 
-By default, Rand is built with all stable features available. The following
+Rand is built with only the `std` feature anabled by default. The following
 optional features are available:
 
--   `alloc` can be used instead of `std` to provide `Vec` and `Box`
--   `i128_support` enables support for generating `u128` and `i128` values
--   `log` enables some logging via the `log` crate
--   `nightly` enables all unstable features (`i128_support`)
--   `serde1` enables serialization for some types, via Serde version 1
--   `stdweb` enables support for `OsRng` on WASM via stdweb.
--   `std` enabled by default; by setting "default-features = false" `no_std`
-    mode is activated; this removes features depending on `std` functionality:
-    -   `OsRng` is entirely unavailable
-    -   `JitterRng` code is still present, but a nanosecond timer must be
-        provided via `JitterRng::new_with_timer`
-    -   Since no external entropy is available, it is not possible to create
-        generators with fresh seeds (user must provide entropy)
-    -   `thread_rng`, `weak_rng` and `random` are all disabled
-    -   exponential, normal and gamma type distributions are unavailable
-        since `exp` and `log` functions are not provided in `core`
-    -   any code requiring `Vec` or `Box`
-
-## Testing
-
-Unfortunately, `cargo test` does not test everything. The following tests are
-recommended:
-
-```
-# Basic tests for Rand and sub-crates
-cargo test --all
-
-# Test no_std support
-cargo test --tests --no-default-features
-# Test no_std+alloc support
-cargo test --tests --no-default-features --features alloc
-
-# Test log and serde support
-cargo test --features serde1,log
-
-# Test 128-bit support (requires nightly)
-cargo test --all --features nightly
-
-# Benchmarks (requires nightly)
-cargo bench
-# or just to test the benchmark code:
-cargo test --benches
-```
+- `alloc` can be used instead of `std` to provide `Vec` and `Box`.
+- `i128_support` enables support for generating `u128` and `i128` values.
+- `log` enables some logging via the `log` crate.
+- `nightly` enables all unstable features (`i128_support`).
+- `serde1` enables serialization for some types, via Serde version 1.
+- `stdweb` enables support for `OsRng` on WASM via stdweb.
+
+`no_std` mode is activated by setting `default-features = false`; this removes
+functionality depending on `std`:
+
+- `thread_rng()`, and `random()` are not available, as they require thread-local
+  storage.
+- `OsRng` is unavailable.
```

Comment:
and `EntropyRng`

Is it worth linking to #313 somewhere?

Reply:
We could. Is it important enough for the readme?